web-marketplace/src/components/organisms/SettingsForm/SettingsForm.schema.ts

```diff
@@ -1,7 +1,8 @@
+import { getIsUuid } from 'components/templates/ProjectDetails/ProjectDetails.utils';
 import { z } from 'zod';
 
 export const settingsFormSchema = z.object({
-  slug: z.string(),
+  slug: z.string().refine(value => !getIsUuid(value), { message: `Slug can't be set to a UUID` }),
```

Comment:
In order to avoid edge cases where a slug would be set to the project's UUID or another project's UUID, we just make sure to reject any slug that is in the form of some UUID. This doesn't really make sense to set a slug as a UUID anyway so I believe that's fine.